src/main/java/edu/cornell/kfs/module/purap/document/service/impl/PurchaseOrderTransmissionMethodDataRulesServiceImpl.java

```diff
@@ -16,14 +15,16 @@
  */
 public class PurchaseOrderTransmissionMethodDataRulesServiceImpl implements PurchaseOrderTransmissionMethodDataRulesService{
 	
+	private PostalCodeValidationService postalCodeValidationService;

```

Comment:
In theory but not necessarily in practice. I'd have to dig up the details, but I know IU submitted contributions to change some of them to protected, and I think we generally did that on the product team. I think it may have to do with spring, beanutils, reflection, etc. Some things don't actually use the setters & getters.